provider/cmd/pulumi-resource-pulumiservice/schema.json

```
@@ -483,6 +483,31 @@
           "value": "destroy"
         }
       ]
+    },
+    "pulumiservice:index:EnvironmentPermission": {
+      "type": "string",
+      "enum": [
+        {
+          "description": "No permissions.",
+          "value": "none"
+        },
+        {
+          "description": "Permission to read unopened environment only.",
```

nit: we mostly refer to this as the environment definition. "unopened" is a nice alternative but I think we should prioritize consistency here

```suggestion
          "description": "Permission to read environment definition only.",
```